pagopa/io-app
# chore(IT Wallet): [SIW-1380] Add decoders for image without url schema and MDL privileges

## Short description
This PR introduces two new decoders for a credential claim, one for an image in base64 without URL schema (due to an issue from the issuer which doesn't provide a URL scheme) and another one for the MDL privileges.

## List of changes proposed in this pull request
- Add `ItwCodecUtils.ts` which is a copy of [this](https://github.com/gcanti/io-ts-types/blob/master/src/JsonFromString.ts) decoder for parsing a JSON from a string;
-Add `ImageClaimNoUrl` decoder for decoding a base64 without URL scheme;
- Update `DrivingPrivilegesClaim` which is now provided as a JSON string which contains an array of objects with a different schema from the one provided in the PoC. It's a combination of a string, json from string and array of object decoders.
- Update `ItwCredentialClaim` with the aformentioned decoders along with the new design for the MDL privileges; 
- Update `ItwRequiredClaimsList`  with the aformentioned decoders and support for multiple claims in a single one like the MDL privileges;

Ignore the large number of commits, they are already in `master`.

## How to test
Try to obtain a MDL and check if MDL privileges and image are properly rendered in the requested claims list, preview and detail screens.
